File: 351001/Shimanchuk/RV2/src/main/java/com/bsuir/romanmuhtasarov/domain/response/NewsResponseTo.java
package com.bsuir.romanmuhtasarov.domain.response;

import java.time.LocalDateTime;
import java.util.List;

public record NewsResponseTo(
        Long id,
        Long writerId,
        String title,
        String content,
        LocalDateTime created,
        LocalDateTime modified, //   List<TagResponseTo> stickerList
        List<CommentResponseTo> commentList) {
}

File: 351003/Kisel/Rest/Task310RestApi/Task310RestApi/DTOs/Response/NewsResponseTo.cs
using Newtonsoft.Json;

namespace Task310RestApi.DTOs.Response
{
    public class NewsResponseTo
    {
        [JsonProperty("id")]
        public long Id { get; set; }

        [JsonProperty("creatorId")]
        public long CreatorId { get; set; }

        [JsonProperty("title")]
        public string Title { get; set; } = string.Empty;

        [JsonProperty("content")]
        public string Content { get; set; } = string.Empty;

        [JsonProperty("created")]
        public DateTime Created { get; set; }

        [JsonProperty("modified")]
        public DateTime Modified { get; set; }

        [JsonProperty("labelIds")]
        public List<long> LabelIds { get; set; } = new List<long>();
    }
}